src/Parser/Rule/Source.idr

```diff
@@ -46,6 +46,10 @@ constant
                            Ident "Char"    => Just CharType
```

Comment:
Should this whole block just be
```idris
                           Ident str       => isConstantType (UN str)
```
? Not sure whether the extra `"%World"  => Just WorldType` is problematic or not.

Reply:
Done. Made sure, `WorldType` would not be returned (same behavior as before).